catalog/transformer/catalog-transformer-service-atom/src/main/java/ddf/catalog/transformer/response/query/atom/AtomTransformer.java

```diff
@@ -454,7 +455,7 @@ private Link addLinkHelper(Action action, Entry entry, String linkType, String m
           if (geo != null) {
 
             try {
-              Geometry geometry = reader.read(geo.toString());
+              Geometry geometry = new WKTReader(GEOMETRY_FACTORY).read(geo.toString());
```

Comment:
❓ the thread safety is more important, but just curious if there were any performance concerns with constructing the new WKTReader each time `setViewMetacardActionProvider` is called?

Reply:
In other areas, we wrap it in ThreadLocal and make it a global to avoid having to create a new WKTReader on every invocation of a method.

Reply:
We could use threadLocal... but changing that here doesn't get it in to master where this is merged.  We reused the Geometry Factory and creating a WKTReader itself is an inexpensive simple object.